### PR TITLE
chore: added four resource links

### LINKED ISSUE #679: [Add] Phind, The AI search engine for developers
### Site URL

https://www.phind.com/

### Description of the above provided site

Phind is a search engine that simply tells users what the answer is. Optimized for developers and technical questions, Phind instantly answers questions with detailed explanations and relevant code snippets from the web.

### Category

Other

### Sub Category

Dev Tools

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### LINKED ISSUE #680: [Add] Awesome Certificates for showcase
### Site URL

https://github.com/PanXProject/awesome-certificates

### Description of the above provided site

A curated of list of IT, computer science and business courses with certificates & digital badges to earn upon completion.

Each course on the list is free to take as well as the badge and certificate.

### Category

Other

### Sub Category

Certifications

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### LINKED ISSUE #681: [Add] List of Learning resources for programmers
### Site URL

https://rentry.co/ProgrammingLearningResources

### Description of the above provided site

List containing hundreds of best learning resources available for programmers.
Categorised in languages and technologies for easy navigation.

### Category

Other

### Sub Category

websites

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### LINKED ISSUE #683: [Add] Rip Tutorial a website for full path map for languages.
### Site URL

https://riptutorial.com/

### Description of the above provided site

Archived Stack Overflow Documentation
RIP Tutorial is a practical free learning tool. 
Find plenty of information available.
Website contains plenty of eBook resources and examples for learning help to developers.

### Category

Others

### Sub Category

websites

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by dhrumilpatel30**
## Fixes Issue

Closes #679 
Closes #680 
Closes #681 
Closes #683

## Changes proposed

added the following resources:

RIP Tutorial
best Resource Collection
Awesome Certificates list
AI search engine for developers

## Screenshots

image

## Note to reviewers

as per instructed I have worked on four addition of a link in one pull request, if you find it difficult to manage I will make each PR for each issue listed above.
Thanks

### CODE REVIEW

File: database/Other/websites.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+[
+    {
+        "name": "RIP Tutorial",
+        "description": "RIP Tutorial is a practical free learning tool. Website contains plenty of eBook resources and examples for learning help to developers.",
```
Comment (CBID2) [inline database/Other/websites.json:4]: [Proofreading]
```suggestion
        "description": "RIP Tutorial is a free website that contains plenty of eBook resources and examples that can be helpful to developers.",
```

File: database/Other/websites.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        "url": "https://riptutorial.com/",
+        "category": "Other",
+        "subcategory": "websites"
+    },
+    {
+        "name": "Resource Collection",
+        "description": "List containing hundreds of best learning resources available for programmers. Categorised in languages and technologies for easy navigation.",
```
Comment (CBID2) [inline database/Other/websites.json:11]: [Proofreading]
```suggestion
        "description": "This is a list containing the best learning resources available for programmers. It is categorized by language and technology for easy navigation.",
```

File: database/Other/websites.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        "url": "https://rentry.co/ProgrammingLearningResources",
+        "category": "Other",
+        "subcategory": "websites"
+    },
+    {
+        "name": "Awesome Certificates",
+        "description": "A curated of list of IT, computer science and business courses with certificates & digital badges to earn upon completion. Each course on the list is free to take as well as the badge and certificate.",
```
Comment (CBID2) [inline database/Other/websites.json:18]: [Proofreading]
```suggestion
        "description": "A curated of list of IT, computer science, and business courses with certificates & digital badges to earn upon completion. Each course on the list is free.",
```

File: database/Other/websites.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        "url": "https://github.com/PanXProject/awesome-certificates",
+        "category": "Other",
+        "subcategory": "websites"
+    },
+    {
+        "name": "AI search engine for developers",
+        "description": "Phind is a search engine that simply tells users what the answer is. Optimized for developers and technical questions.",
```
Comment (CBID2) [inline database/Other/websites.json:25]: [Proofreading]
```suggestion
        "description": "Phind is a search engine that simply tells users what the answer is. It is optimized for developers and technical questions.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @dhrumilpatel30! I just gave some feedback on how to improve your descriptions. As I have said in your other PRs, run your descriptions through Grammarly or Spellcheck to ensure that they are clear.

**Review by dhrumilpatel30 — COMMENTED** [review]
thanks I have made changes according to your suggestions

**Review by rupali-codes — CHANGES_REQUESTED** [review]
* Since RIP tutorial is an e-book, it could be placed under _resources/e-book_ category
* AI search engine, could be added in _others/dev-tools_ category

And for Resources collection and Awesome certificates, you rename this subcategory from **websites** to **other resources**

**Comment by dhrumilpatel30** [comment]
> * Since RIP tutorial is an e-book, it could be placed under _resources/e-book_ category
> * AI search engine, could be added in _others/dev-tools_ category
> 
> And for Resources collection and Awesome certificates, you rename this subcategory from **websites** to **other resources**

hi @rupali-codes, thanks for review. I have made changes hope it will solve it.

**Review by rupali-codes — CHANGES_REQUESTED** [review]
otherresources getting hard to read, could you make it _other-resources_

**Comment by dhrumilpatel30** [comment]
> otherresources getting hard to read, could you make it _other-resources_

hi @rupali-codes, I have just changed it. please suggest if any more changes needed.

**Comment by CBID2** [comment]
Hey, @dhrumilpatel30. Can you resolve these branch conflicts, please?

**Comment by dhrumilpatel30** [comment]
> Hey, @dhrumilpatel30. Can you resolve these branch conflicts, please?

yes sure

**Comment by dhrumilpatel30** [comment]
hi @CBID2, I have resolved conflicts.

**Review by rupali-codes — APPROVED** [review]
looks good

**Comment by rupali-codes** [comment]
@CBID2 could you please review this PR?

**Review by CBID2 — APPROVED** [review]
LGTM!

**Comment by CBID2** [comment]
> @CBID2 could you please review this PR?

Done @rupali-codes

**Review by rupali-codes — APPROVED** [review]
lgtm too

**Pull request merged**